recipes/sctools/meta.yaml

```diff
@@ -17,8 +17,6 @@ requirements:
   build:
     - {{ compiler('cxx') }}
     - cmake >=3.0.0
-    - seqan-library
-    - zlib
```

Comment:
We only need build tools in `requirements/build` and can put all libraries that we may link to in `requirements/host`.
And it's correct that we don't need the `requirements/run` section, because
- `zlib` should have a `run_export` so that we get a `run` dependency automatically,
- SeqAn is header-only (unless that changed, it has been a couple of years since I used it last) and doesn't impose an additional `run` requirement.